tests/background/uri-info-test.js

```diff
@@ -1,72 +1,113 @@
 import * as uriInfo from '../../src/background/uri-info';
 import { toResult } from '../promise-util';
 import settings from '../settings.json';
-import { unroll } from '../util';
 
-describe('UriInfo.query', function () {
-  var badgeURL = settings.apiUrl + '/badge';
+describe('UriInfo', () => {
```

Comment:
```suggestion
describe('background/uri-info', () => {
```

This would be more inline with our current conventions, since we're not testing a class or function named `UriInfo`.